modules/ytp-assets-common/src/less/ckan/ckan.less

```diff
@@ -130,10 +130,14 @@
 .btn-success {
   background-color: @btn-dark-bg;
   border-color: @btn-dark-bg;
-  &:hover {
+  &:hover, &:focus, &:active, &.disabled:focus, &.disabled:hover {
```

Comment:
The disabled color should be clearly different from active hover or normal hover.

Reply:
I changed the color for disabled class